bodo/pandas/frame.py

```diff
@@ -866,6 +866,11 @@ def groupby(
         Provides support for groupby similar to Pandas:
         https://github.com/pandas-dev/pandas/blob/0691c5cf90477d3503834d983f69350f250a6ff7/pandas/core/frame.py#L9148
         """
+        if sort != False:
+            raise BodoLibNotImplementedException(
+                "groupby: only sort=False currently supported"
+            )
```

Comment:
Requiring the user to always set sort=False to use Bodo is a significant burden I believe. This is a well-known issue of Pandas APIs when parallelized since users rarely require the sort but is expensive in parallel. As a pragmatic solution, we can just set `sort=False` by default same as JIT: https://docs.bodo.ai/latest/api_docs/pandas/dataframe/groupby/

Reply:
Have we already accepted other deviations from Pandas standards for default values?  If so, I think this breaks the "one line change" mantra.  It is a small thing for sure but not something I agree with.  Nevertheless, I've reverted it based on your preference.

Reply:
resolved